tests/scripts/thread-cert/run_cert_suite.py

```diff
@@ -39,7 +39,7 @@
 
 THREAD_VERSION = os.getenv('THREAD_VERSION')
 VIRTUAL_TIME = int(os.getenv('VIRTUAL_TIME', '1'))
-MAX_JOBS = multiprocessing.cpu_count() * 2 if VIRTUAL_TIME else 10
+MAX_JOBS = int(os.getenv('MAX_JOBS', (multiprocessing.cpu_count() * 2 if VIRTUAL_TIME else 10)))
```

Comment:
is the environment variable just for local use?

Reply:
yes.

Reply:
Prefer adding a `OT_` prefix for new such kind of environments, thoughts?
```suggestion
MAX_JOBS = int(os.getenv('OT_MAX_JOBS', (multiprocessing.cpu_count() * 2 if VIRTUAL_TIME else 10)))
```

Reply:
Note that we don't have `OT_` prefix for `VIRTUAL_TIME` and `THREAD_VERSION`.
I prefer `MAX_JOBS` without `OT_` because it's just for local debugging (not to be used by general OT users).

Reply:
Maybe we can have `OT_TEST_` prefix for test variables, i.e. `OT_TEST_MAX_JOBS`